flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/RowDataSerializer.java

```diff
@@ -40,12 +53,18 @@
  * Serializer for RowData.
  */
 public class RowDataSerializer implements DorisRecordSerializer<RowData> {
```

Comment:
It is recommended to write an arrow ut in TestRowDataSerializer

Reply:
done